### ISSUE TITLE
Train a new model starting from pre-trained COCO weights

### ISSUE THREAD

**Issue by blossom2016** [matterport/Mask_RCNN#2043]
I dont get how this works. I am new to ML so i will need help with this. I use a windows system

**Comment by suchiz** [comment]
Follow the balloon example. It it very well directed even thought if you're new. It explains everything :).

https://engineering.matterport.com/splash-of-color-instance-segmentation-with-mask-r-cnn-and-tensorflow-7c761e238b46
